MDANSE_GUI/Src/MDANSE_GUI/Tabs/TrajectoryTab.py

```diff
@@ -60,9 +60,13 @@ def load_trajectories(self):
             str(self.get_path("trajectory")),
             "HDF5 files, MDANSE or H5MD format (*.mdt *.h5);;H5MD files (*.h5);;All files (*)",
         )
+
+        loaded_files = []  # list to store loaded files for recent files tracking
+
         for fname in fnames[0]:
             self.load_trajectory(PurePath(fname))
             last_path = str(PurePath(os.path.split(fname)[0]))
+            loaded_files.append(str(PurePath(fname)))
```

Comment:
```suggestion
        loaded_files = []  # list to store loaded files for recent files tracking

        for fname in fnames[0]:
            filepath = Path(fname)
            self.load_trajectory(filepath)
            last_path = str(filepath.parent)
            loaded_files.append(str(filepath))
```
Using `Path`s effectively will be easier.